script/DeployHypoVaultArchitectureEoa.s.sol

```diff
@@ -53,218 +34,30 @@ contract DeployHypoVaultArchitectureEoa is Script, MerkleTreeHelper {
         console.log("Factory Address:", vaultFactoryAddress);
 
         // 3. Deploy new Accountant with CREATE2
-        PanopticVaultAccountant accountant = new PanopticVaultAccountant{salt: salt}(owner);
+        PanopticVaultAccountant accountant = new PanopticVaultAccountant{salt: salt}(deployer);
         address accountantAddress = address(accountant);
         console.log("=== CREATE2 Deployment Info ===");
         console.log("Accountant Address:", accountantAddress);
 
-        // 4. Deploy HypoVault via Factory
-        uint256 performanceFeeBps = 1000; // 10%
-        address wethPlpVaultAddress = vaultFactory.createVault(
+        // 4. Deploy WETH vault
+        deployVault(
```

Comment:
Sorry, I mispoke in the [first AC here](https://github.com/orgs/panoptic-labs/projects/17/views/6?pane=issue&itemId=143442922&issue=panoptic-labs%7Chypovault%7C14). What i meant is there should 1 architecture deployment script which deploys all the contracts used for all vaults on a given chain, and then 1 script per vault.

So please move this WETH vault deployment out into it's own script like you did the USDC script. Even though it's already deployed on Sepolia, we'll need it on it's own script for mainnet deployment organization